staging/src/k8s.io/kube-aggregator/pkg/apiserver/metrics.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apiserver
+
+import (
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+var x509MissingSANCounter = metrics.NewCounter(
+	&metrics.CounterOpts{
```

Comment:
no labels? How an admin could identify a serving cert without SAN?

Reply:
There was a lengthy discussion in https://github.com/kubernetes/kubernetes/pull/85480 about certificates and PI. The outcome infomation you might use for a label would, in this case, have informational value close to having no labels at all.

Also, I am not willing to perform any new changes to this PR that might bring sig/instrumentation into the loop for review again, getting their review proved to be near to impossible.